src/caret_analyze/architecture/architecture.py

```diff
@@ -377,13 +377,35 @@ def assign_variable_passings(self, node_name: str,
                               AssignContextReader(node)))
 
     def rename_callback(self, src: str, dst: str) -> None:
+        """
+        Update callback name "src" to "dst" in architecture.
+
+        Parameters
+        ----------
+        src : str
+            callback name to be updated
+        dst : str
+            callback name to be updating
```

Comment:
These may be a nuance which is difficult to interpret.
```suggestion
        src : str
            current callback name
        dst : str
            updated callback name
```

Reply:
Applied in [96d7b4e](https://github.com/tier4/CARET_analyze/pull/253/commits/96d7b4e912b35325087313cbd627b4d4ad8ba4be).
Thank you for suggestion!

Reply:
@keita1523 
Applied in [96d7b4e](https://github.com/tier4/CARET_analyze/pull/253/commits/96d7b4e912b35325087313cbd627b4d4ad8ba4be).
Thank you for suggestion!